docs/proposals/20190610-machine-states-preboot-bootstrapping.md

```diff
@@ -360,9 +360,9 @@ MachinePhaseFailed = MachinePhaseType("failed")
 #### Expectations
 - User intervention.
 
-![Figure 1](./images/machine-states-preboot/Figure1.png)
-
 ---
+![Figure 1](../book/src/images/bootstrap-controller.png)
```

Comment:
TBH, I'm not really sure we should mix images from the book and proposal images, because this is potentially error prone.

Instead, I will be in favour of duplicating the image and keep one copy under `/images/machine-states-preboot`, thoghether with all the images for this proposal.

But I'm happy to adapt to other opinions if someone has strong preferences.

Reply:
We should probably do the other way around? Embed proposal images in the book?

Reply:
I'm more inclined to duplicate the image and keep one copy under `/images/machine-states-preboot`,  pr #1318 seems to be no consideration of this image being used in `/images/machine-states-preboot`?

Reply:
If we go down that path, we need to make sure that images are updated in both places and kept in sync. Is there any issue to link to images in the proposals folder from the book?

Reply:
Maybe the question is if a link leads to implicit changes in the documentation when updating proposals (not sure how often that happens).

Alternative solution could be to duplicate the image and add some kind of linter which ensures that they are always sync to enforce manual sync of the images and thus the documentation. But not sure if it's really worth the effort.

Reply:
@chymy Are you able to address this?

Reply:
@vincepri what's the action item on this? Seems to me like this PR is fixing something pretty trivial and might not be the best place to bikeshed on proposal/book images.

if there is a clear path forward let's fix it and get it merged though, less broken links == better docs

Reply:
We can merge this fix. Although I would like to find a long term solution given that it seems we're now mixing images between the book and the proposal.